Mediapipe.Net/Calculators/CpuCalculator.cs

```diff
@@ -13,6 +13,11 @@
 
 namespace Mediapipe.Net.Calculators
 {
+    /// <summary>
+    /// This abstract class is the base for any CPU calculator.
+    /// </summary>
+    /// <typeparam name="TPacket">The type of packet the calculator returns the landmarks in.</typeparam>
+    /// <typeparam name="T">The type of landmarks output.</typeparam>
```

Comment:
- "This abstract class is" is redundant.
- "landmarks" is too specific, you might want to use an object detection calculator for instance, which I'm sure doesn't output any type of landmarks.